config/config.go

```diff
@@ -143,9 +143,13 @@ func NewDefaultConfig() *AgentConfig {
 		CMAB: CMABConfig{
```

Comment:
Why ODP is not in ClientConfig?  Ideally should be consistent with ODP?

Reply:
Silly me, you're totally right. They should be together.